src/js/script.js

```diff
@@ -686,10 +242,7 @@ closeButton.addEventListener('click', () => {
 })
 
 //  search
-
-
 const searchField = document.querySelector('.search__input')
-
 let searchResult =''
 
 searchField.addEventListener('input', e => {
```

Comment:
you dont call showSearchResult, its work ??
searchField.addEventListener('input', showSearchResult)
function showSearchResult () {...} - take it higher